test/e2e/support/plugins.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Node dependencies
+ */
+import { visitAdmin } from './utils';
+
+export async function installPlugin( name, searchTerm ) {
```

Comment:
What would be the purpose of the second argument here? Not clear by the included documentation 😉